package.json

```diff
@@ -2,7 +2,7 @@
   "name": "patreon",
   "version": "0.1.0",
```

Comment:
We don't need to bump major (changes to the api) in the PR, but we shouldn't publish to NPM until we do.